core/cloudflow-spark-tests/src/test/scala/cloudflow/spark/SparkEgressSpec.scala

```diff
@@ -81,8 +81,8 @@ class SparkEgressSpec extends SparkScalaTestSupport {
       val r2 = asCollection[String](session, "allNamesUpper")
 
       // assert
-      r1 must contain("name1")
-      r2 must contain("NAME1")
+      r1 must contain("name2")
```

Comment:
This also does not look like it is related to read all partitions. Why was this change necessary? Did the test fail?

Reply:
Done